feg/gateway/services/n7_n40_proxy/servicers/session_controller.go

```diff
@@ -46,7 +50,18 @@ func NewCentralSessionController(
 	n7config *n7.N7Config,
 	dbClient policydb.PolicyDBClient,
 	policyClient n7_sbi.ClientWithResponsesInterface,
+	cloudReg service_registry.GatewayRegistry,
 ) (*CentralSessionController, error) {
+	notifHandler := notify.NewNotificationHandler(&n7config.Client, cloudReg)
+	err := notifHandler.RegisterHandlers()
+	if err != nil {
+		return nil, err
+	}
+	// Start the notification handler HTTP service
+	_, err = notifHandler.Start()
+	if err != nil {
+		return nil, fmt.Errorf("notification handler failed to start - %s", err)
+	}
```

Comment:
As mentioned This logic here should be placed on the NewNotificationHandler.
 SessionController should receive an already working object with all the hanlders mounted